crates/mun_syntax/src/tests/snapshots/parser__function.snap

```diff
@@ -97,15 +98,23 @@ SOURCE_FILE@[0; 158)
               NAME_REF@[141; 147)
                 IDENT@[141; 147) "number"
       R_PAREN@[147; 148) ")"
-    RET_TYPE@[148; 155)
-      COLON@[148; 149) ":"
-      PATH_TYPE@[149; 155)
-        PATH@[149; 155)
-          PATH_SEGMENT@[149; 155)
-            NAME_REF@[149; 155)
-              IDENT@[149; 155) "number"
-    WHITESPACE@[155; 156) " "
-    BLOCK_EXPR@[156; 158)
-      L_CURLY@[156; 157) "{"
-      R_CURLY@[157; 158) "}"
+  ERROR@[148; 149)
+    COLON@[148; 149) ":"
+  ERROR@[149; 155)
+    IDENT@[149; 155) "number"
+  WHITESPACE@[155; 156) " "
+  ERROR@[156; 157)
+    L_CURLY@[156; 157) "{"
+  ERROR@[157; 158)
+    R_CURLY@[157; 158) "}"
+error Offset(120): expected a block
```

Comment:
These errors weren't here before. I think you forgot to change this test.

Reply:
When I run `cargo test --release --all`, all of the tests succeed, and `cargo insta review` says that there are no snapshots to review.

Reply:
Thats correct because I think you forgot to change `:` to `->` in the input so the errors are to be expected. However, they weren't there before. I think you missed this file:
https://github.com/mun-lang/mun/blob/master/crates/mun_syntax/src/tests/parser.rs

Reply:
Fixed.